config/buildspec_zero_code_change.yml

```diff
@@ -30,14 +29,10 @@ phases:
   build:
     commands:
       - cd $CODEBUILD_SRC_DIR && chmod +x config/install_smdebug.sh && chmod +x config/check_smdebug_install.sh && ./config/install_smdebug.sh && cd ..
-      - cd $RULES_CODEBUILD_SRC_DIR && python setup.py bdist_wheel --universal && pip install dist/*.whl && cd ..
-      - cd $CODEBUILD_SRC_DIR  && chmod +x config/tests.sh && PYTHONPATH=. ./config/tests.sh  && mkdir -p upload/$CURRENT_COMMIT_PATH/wheels && cp ./dist/*.whl upload/$CURRENT_COMMIT_PATH/wheels && cd ..
-      - cd $RULES_CODEBUILD_SRC_DIR && chmod +x config/tests.sh && mkdir -p upload/$CURRENT_COMMIT_PATH/wheels && PYTHONPATH=. ./config/tests.sh  && cp ./dist/*.whl upload/$CURRENT_COMMIT_PATH/wheels && cd ..
+      - cd $CODEBUILD_SRC_DIR  && chmod +x config/tests.sh && PYTHONPATH=. ./config/tests.sh
+      - cd $CODEBUILD_SRC_DIR_RULES && chmod +x config/tests.sh
 
   post_build:
     commands:
-      - . $CODEBUILD_SRC_DIR/config/upload_on_end.sh
-      - rm -rf $CODEBUILD_SRC_DIR/upload/$CURRENT_COMMIT_PATH
-      - rm -rf $RULES_CODEBUILD_SRC_DIR/upload/$CURRENT_COMMIT_PATH
```

Comment:
why removed, I guess upload on end also uploads logs

Reply:
the build project for this buildspec is only run as part of the nightly pipeline. So this will anyway get overridden by pipeline settings and all the logs will get uploaded to the s3 bucket specified by the pipeline artifacts. So i thought I will remove it. Can put it back, but it should not make any difference.

Reply:
One use case if we want to run specific tests like TF tests for certain PRs which changes and affects TF ZCC . 
We should have a way to run these test in isolation outside of pipeline, if it manually triggers.

Reply:
ok. will change accordingly.